### DISCUSSION TITLE
Umi 4  .umirc.ts 的 extraBabelPlugins 配置后, TS 报错.

### DISCUSSION THREAD

**Discussion by xxxxue** [umijs/umi#9169] [General]
Umi 4  `.umirc.ts` 的 `extraBabelPlugins` 配置后 TS 报错. 如图
( Jetbrains IDE 使用 defineConfig 包住也是一样报这个错误 )

![image](https://user-images.githubusercontent.com/32764266/187477129-b9f60aef-24e5-43f3-8588-b27b592391cd.png)

图中的代码:  
`(加上 any 后, 功能是完全正常可用的.)`
```ts
extraBabelPlugins: [
    ['import', {
        "libraryName": "antd",
        "libraryDirectory": "lib",
        "style": "css"
    }],
] // as any
```

版本号: 
- Umi Version: v4.0.15
- Node Version: v16.14.2
- Platform: Windows 11
- VSCode: v1.70.0

**Comment by fz6m** [comment]
升级 ts 版本至最新（确保 IDE 的 ts server 引擎正在使用项目内的 typescript ）

**Reply by xxxxue** [reply]
确实是 用的项目内最新的 TS 版本 4.8.2
